test/server/routes/loginPostRoute.js

```diff
@@ -0,0 +1,64 @@
+const test = require('tape');
+const supertest = require('supertest');
+const connect = require('../../../server/database/config/connection');
+const router = require('../../../server/app');
+
+const checkUser =  username => connect.query('select * from users where user_name = $1', [username]);
+
+test('Testing /api/v1/login login route for valid user ', (t) => {
+  const user = {
+    user_name: 'Angham116',
+    password: 'Angh12345678',
+  };
+  checkUser(user.username)
+    .then(() => {
+      supertest(router)
+        .post('/api/v1/login')
+        .send(user)
+        .expect(201)
+        .end((err, res) => {
+          if (err) t.error(err);
+          t.equal(typeof res.body, 'object', ' Return typeof object ');
```

Comment:
This isn't a test, the `res.body` could be any object.
Here it'd make sense if you tested the cookie itself